client/layout/global-sidebar/header.tsx

```diff
@@ -14,15 +14,12 @@ export const GlobalSidebarHeader = () => {
 				<span className="dotcom"></span>
 			</a>
 			<span className="gap"></span>
-			<SidebarSearch tooltip={ translate( 'Jump to...' ) } />
+			<SidebarSearch tooltip={ translate( 'Jump to …' ) } />
```

Comment:
```suggestion
			<SidebarSearch tooltip={ translate( 'Jump to…' ) } />
```
Nit: According to the prototype (https://wpcomnav4.web.app/)